apps/shell/Kconfig

```diff
@@ -58,17 +58,18 @@ config TASH_SCRIPT
 		See "sh" on TASH help command.
 
 config SECURED_TASH
-    bool "Enable Secure shell"
-    depends on NET_SECURITY_TLS
-    default n
+	bool "Enable Secure shell"
+	depends on NET_SECURITY_TLS
+	default n
 
 if SECURED_TASH
 
 config TASH_PASSWORD_SHA256
-    string "SHA256 encrypted password"
-    default "0000"
-    ---help---
-        Enable password to enter TASH.
+	string "SHA256 encrypted password (32 Bytes)"
+	default "0000"
```

Comment:
can we set default 32 bytes here

Reply:
I don't know that we need to add default value here. Because it's password, someone who wants to enable this feature can't use default value.
Could you let me know your opinion? Is it better to add for example?

Reply:
I feel better don't add any default password, we can give example...
if user not set any password then feature can't be used